src/_includes/components/footer.njk

```diff
@@ -16,7 +16,7 @@
           <div class="field--connected-btn">
             <input id="newsletter-email" type="email" name="email" 
               placeholder="name@example.com" autocapitalize="off" required />
-            <input type="hidden" name="tags" value="Web Sign up"/>
```

Comment:
We still need this. We do want to send tags along to Buttondown

Reply:
Fixed!